src/uint/modular/runtime_mod.rs

```diff
@@ -59,6 +61,28 @@ impl<const LIMBS: usize> DynResidueParams<LIMBS> {
         }
     }
 
+    /// Instantiates a new set of `ResidueParams` representing the given `modulus`, which _must_ be odd.
+    /// If `modulus` is not odd, this function will panic; use [`new_checked`][`DynResidueParams::new_checked`] if you want to be able to detect an invalid modulus.
+    #[deprecated(
```

Comment:
On second thought, I think a deprecation here might be unhelpful.

We want people to use `new` in the next release, and getting them to rename to `new_checked` probably won’t help there.

So maybe they should just keep `new` for now and update the signature in the next release.

Reply:
Should we keep `new_checked` in this release anyway, to give users the `CtOption` functionality until the signature change? If so, should `new_checked` be immediately deprecated to provide information about the change?

Reply:
We could potentially keep it as a deprecated alias for `new` in the next breaking release.

Otherwise yes, the purpose of giving it a different name is so we can get it into this release.

Reply:
Sure thing. I'll move the deprecation to `new_checked` instead, so users who want more feedback on modulus validity can have that functionality right away. Will do the same in #243.